### ISSUE TITLE
Multiple copies of the OpenMP runtime

### ISSUE THREAD

**Issue by pappuyadav** [ultralytics/ultralytics#543]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Other

### Bug

**Using YOLOv8 directly in a Python environment throws the following error:**
_OMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/._ 
Adding the following two lines  to the existing Python script solves the problem. [Spyder 5.3.3, Python 3.9.16]
import os
os.environ['KMP_DUPLICATE_LIB_OK']='True'
 Here is the modified script that works without the above error.

from ultralytics import YOLO
import os
os.environ['KMP_DUPLICATE_LIB_OK']='True'
# Load a model
model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)
# Use the model
results = model.train(data="coco128.yaml", epochs=3)  # train the model
results = model.val()  # evaluate model performance on the validation set
results = model("https://ultralytics.com/images/bus.jpg")  # predict on an image
success = model.export(format="onnx")  # export the model to ONNX format

### Environment

-OS Windows 11 Pro
-Python 3.9.16
-Spyder 5.3.3

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by evia1** [comment]
Hi,
There is an open topic with subject under yolov5:
https://github.com/ultralytics/yolov5/issues/5086

Perhaps you can find your answer there

**Comment by pderrenger** [comment]
@evia1 hi there! It looks like you might have mistaken this repository for YOLOv5, which is a separate repository from YOLOv8. However, I'd be happy to assist you with the issue you're encountering with YOLOv8. 

The error you're experiencing is related to the OpenMP runtime and can be addressed by setting the environment variable `KMP_DUPLICATE_LIB_OK` to `'True'`, as you mentioned. This allows the program to continue executing without the error. 

It's great to see that you've already found a solution to the problem. If you have any further questions or need assistance with anything else related to YOLOv8, feel free to ask! 😊
